assets/quick-add.css

```diff
@@ -0,0 +1,200 @@
+.quick-add {
+  position: relative;
+  align-self: flex-end;
+  grid-row-start: 4;
+  margin: 0 1.3rem 1rem;
+  z-index: 1;
+}
+
+.quick-add-modal {
+  box-sizing: border-box;
+  opacity: 0;
+  position: fixed;
+  visibility: hidden;
+  z-index: -1;
+  margin: 0 auto;
+  top: 0;
+  left: 0;
+  width: 100%;
+  background: rgba(var(--color-foreground), 0.2);
+  height: 100%;
+}
+
+.quick-add-modal[open] {
+  opacity: 1;
+  visibility: visible;
+  z-index: 101;
+}
+
+.quick-add-modal__content {
+  border-radius: var(--popup-corner-radius);
+  background-color: rgb(var(--color-background));
+  margin: 0 auto;
+  left: 50%;
+  transform: translateX(-50%);
+  margin-top: 3.2rem;
+  width: 100%;
+  position: absolute;
+  top: 0;
+  border-color: rgba(var(--color-foreground), var(--popup-border-opacity));
+  border-style: solid;
+  border-width: var(--popup-border-width);
+  box-shadow: var(--popup-shadow-horizontal-offset) var(--popup-shadow-vertical-offset) var(--popup-shadow-blur-radius) rgba(var(--color-shadow), var(--popup-shadow-opacity));
+  overflow: hidden;
```

Comment:
Here I think we have an issue when there are many variants to pick from or a pretty small screen: 
![](https://screenshot.click/21-03-3sih0-5o6bv.png)

Reply:
I think this is now fixed by setting `overflow-y: auto`. Let me know if it works.